lib/cesiumGeometryToGltfPrimitive.js

```diff
@@ -18,20 +18,32 @@ function getFirstAttributeSemantic(gltf, primitive, semantic, packedLength) {
     var type;
     if (semantic === 'TEXCOORD') {
         type = 'VEC2';
-    } else if (semantic === 'POSITION' || semantic === 'NORMAL') {
+    } else if (semantic.indexOf('POSITION') === 0 ||
+               semantic.indexOf('NORMAL')   === 0 ||
+               semantic.indexOf('TANGENT')  === 0 ||
+               semantic.indexOf('BINORMAL') === 0   ) {
         type = 'VEC3';
     } else {
         throw new DeveloperError('Unsupported attribute semantic: ' + semantic);
     }
     if (semantics.length <= 0) {
-        primitive.attributes[semantic] = createAccessor(gltf, packedLength, type, WebGLConstants.FLOAT, WebGLConstants.ARRAY_BUFFER);
+
+        var id = undefined;
+        
+        // all attributes but texcoords may occur only once
+        if (semantic != 'TEXCOORD') {
```

Comment:
This won't pass JSHint.

Use `!==` for exact comparison without type coercion.